Logger/BookRecord.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Logger;
+
+public record class BookRecord ( string Author, string Title, int Isbn) : BaseEntity(Guid.NewGuid())

```

Comment:
We know it's a record because it is a record class, keeping it just as like `Book` may be simpler and more clear to the object itself. What is a `BookRecord`? But a `Book` is a thing, record is just the type that it's being stored by
```suggestion
public record class Book ( string Author, string Title, int Isbn) : BaseEntity(Guid.NewGuid())

```